gen/test-html.hs

```diff
@@ -21,23 +22,68 @@ data Presentation = Presentation
     { title :: String
     , author :: String
     , tags :: [Tag]
-    , slides :: Maybe String
-    , audio :: Maybe String
-    , player :: Maybe String
+    , slides :: Maybe FilePath
```

Comment:
While I see that `String` is probably not the best type for this, I'm not sure `FilePath` is better. It does not have to be relative path, it can also be absolute url to slides (even though storing presentation locally is preffered).